app/views/effective/learndash_courses/_learndash_course.html.haml

```diff
@@ -4,7 +4,7 @@
       %th Regular Price
       %td= price_to_currency(learndash_course.regular_price)
     %tr
-      %th Member Price
+      %th Discounted Price
```

Comment:
cab doesn't have membres, but they do have a discounted price for existing registrants. but i think this partial isn't gonna be visible anymore soooo